### ISSUE TITLE
4.5.4 not working

### ISSUE THREAD

**Issue by dc-me** [100apps/charles-hacking#37]
#36 this path doesn't work as well.

@100apps @piratesephiroth @HeIp

**Comment by HeIp** [comment]
@danywheeler I'll look into it sometime this weekend, or earlier if I have the time.
I didn't handle the 4.5.1 but I'll look into 4.5.4

**Comment by HeIp** [comment]
@danywheeler & everyone
https://github.com/8enet/Charles-Crack
Is where I have moved to, this process is now simpler 
and there is a website to download the jar automatically enjoy everyone.
At this time version 4.5.6 was patched there.

**Comment by duraki** [comment]
@HeIp repo removed, any other repo for newer cracks?

Forget it, I wrote a keygen: https://github.com/duraki/charles-keygen

**Comment by HeIp** [comment]
Yeah removed 14 days ago prior to today, was very sad. But the live website works.
Site(In Chinese): https://www.zzzmode.com/mytools/charles/
Site(IN English): https://translate.google.com/translate?sl=auto&tl=en&u=https%3A%2F%2Fwww.zzzmode.com%2Fmytools%2Fcharles%2F
My Fork(Works for 4.5.6): https://github.com/HeIp/Charles-Crack
Cool Keygen though 💯

**Comment by duraki** [comment]
Thank you for quick response. Nice web generator tho!

**Comment by HeIp** [comment]
It was by the original repo creator, he kept the website.

**Comment by duraki** [comment]
Ah, gotcha! Good shotout; respect the scene
